zk/client_test.go

```diff
@@ -56,6 +56,15 @@ func (s *ZKClientTestSuite) TearDownTest() {
 	}
 }
 
+func (s *ZKClientTestSuite) TestEmbeddedZk() {
+	err := EnsureZookeeperUp(s.EmbeddedZkPath)
+	s.NoError(err)
+	err = StopZookeeper(s.EmbeddedZkPath)
+	s.NoError(err)
+	err = EnsureZookeeperUp(s.EmbeddedZkPath)
```

Comment:
is this to test reboot? should we stop in the end?

Reply:
Just want to make sure the cluster can start after stop

Reply:
I was wondering is it a good practice to always stop after start. something to think about, non-blocking.